hal.c

```diff
@@ -194,4 +221,24 @@ void hal_ext_uart(uint8_t active)
         USCI_A_UART_disableInterrupt(UART_BRIDGE, USCI_A_UART_RECEIVE_INTERRUPT);
     }
 }
+
+void hal_ext_i2c(void){
```

Comment:
Why is this in two places?

Reply:
Because the code compatible ***launchpad msp430f5529*** and  ***BeagleConnect*** two board.  I just continue the design from @statropy. I didn't test the Launchpad MSP430F5529, and I'm not sure if its I2C pin is consistent with the MSP430F5503. So I put it in two positions to facilitate the modification of MSP430F5529 in the future.